api/groups.go

```diff
@@ -26,8 +26,11 @@ const (
 	GroupsDeletePermission = "groups:delete"
 )
 
+const SortOrderAsc = "asc"
+const SortOrderDesc = "desc"
```

Comment:
Do these constants need to be exported? If they are private, change to lowercase initial letter. Otherwise add godoc.
```suggestion
const (
    sortOrderAsc = "asc"
    sortOrderDesc = "desc"
)
```